evergreen/eg_app/tests/util/test_validators.py

```diff
@@ -1,5 +1,5 @@
 from django.test import TestCase
-from eg_app.util.validators import validate_email, validate_password
+from eg_app.eg_util.validators import validate_email, validate_password
```

Comment:
The `eg_app/tests/*` subdirectories should mirror `eg_app/*`. If `eg_app/util` is renamed, `eg_app/tests/eg_util` should also be updated